lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -3413,7 +3413,7 @@ size_t TypeSystemSwiftTypeRef::GetIndexOfChildMemberWithName(
       auto found_numidx = runtime->GetIndexOfChildMemberWithName(
           GetCanonicalType(type), name, exe_ctx, omit_empty_base_classes,
           child_indexes);
-      if (found_numidx.first) {
+      if (found_numidx.first != SwiftLanguageRuntime::eError) {
```

Comment:
Shouldn't this also be different than `SwiftLanguageRuntime::eNotFound` too? So maybe just check if `found_numidx.first == SwiftLanguageRuntime::eFound`?